docs/source/API/core/policies/TeamPolicy.rst

```diff
@@ -0,0 +1,239 @@
+| LaunchBounds   | ``LaunchBounds<MaxThreads, MinBlocks>``                                    | Specifies hints to to the compiler about CUDA/HIP launch bounds.                                                                                        |
++----------------+----------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| WorkTag        | ``SomeClass``                                                              | Specify the work tag type used to call the functor operator. Any arbitrary type defaults to ``void``.                                                   |
++----------------+----------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Public Class Members
+--------------------
+
+Constructors
+~~~~~~~~~~~~
+ 
+.. cppkokkos:function:: TeamPolicy()
+   
+    * Default Constructor uninitialized policy.
+
+.. cppkokkos:function:: TeamPolicy(index_type league_size, index_type team_size, index_type vector_length=1)
+
+    * Request to launch ``league_size`` work items, each of which is assigned to a team of threads with ``team_size`` threads, using a vector length of ``vector_length``. If the team size is not possible when calling a parallel policy, that kernel launch may throw. 
+
+.. cppkokkos:function:: TeamPolicy(index_type league_size, Impl::AUTO_t, index_type vector_length=1)
+
+    * Request to launch ``league_size`` work items, each of which is assigned to a team of threads of a size determined by Kokkos, using a vector length of ``vector_length``. The team size may be determined lazily at launch time, taking into account properties of the functor.
+
+.. cppkokkos:function:: TeamPolicy(execution_space space, index_type league_size, index_type team_size, index_type vector_length=1)
+
+    * Request to launch ``league_size`` work items, each of which is assigned to a team of threads with ``team_size`` threads, using a vector length of ``vector_length``. If the team size is not possible when calling a parallel policy, that kernel launch may throw. Use the provided execution space instance during a kernel launch.
+
+.. cppkokkos:function:: TeamPolicy(execution_space space, index_type league_size, Impl::AUTO_t, index_type vector_length=1)
+
+    * Request to launch ``league_size`` work items, each of which is assigned to a team of threads of a size determined by Kokkos, using a vector length of ``vector_length``. The team size may be determined lazily at launch time, taking into account properties of the functor. Use the provided execution space instance during a kernel launch.
+
```

Comment:
in the constructors above, we are missing the copy constructor and move constr, and destructor (see synopsis)

Reply:
Copy constructor ok
Move constructor ok (see screenshots)

I didn't find the destructor... neither in the documentation nor in the source code... it's something like this ~NameOfTheClass(), no ?